jsk_perception/sample/sample_people_pose_estimation_2d.launch

```diff
@@ -2,30 +2,70 @@
 
   <arg name="gui" default="true" />
   <arg name="gpu" default="-1" />
+  <arg name="coral" default="false" />
   <arg name="INPUT_IMAGE" default="/camera/rgb/image_rect_color" />
   <arg name="INPUT_DEPTH_IMAGE" default="/camera/depth_registered/hw_registered/image_rect_raw" />
   <arg name="INPUT_CAMERA_INFO" default="/camera/rgb/camera_info" />
-  <arg name="LIMB_PART" default="RHand" />
+  <arg name="LIMB_PART" default="right wrist" if="$(arg coral)"/>
+  <arg name="LIMB_PART" default="RHand" unless="$(arg coral)"/>
 
   <include file="$(find jsk_perception)/sample/include/play_rosbag_people.xml" />
 
-  <node name="people_pose_estimation_2d"
-        pkg="jsk_perception" type="people_pose_estimation_2d.py"
-        output="screen">
-    <remap from="~input" to="$(arg INPUT_IMAGE)" />
-    <remap from="~input/info" to="$(arg INPUT_CAMERA_INFO)" />
-    <remap from="~input/depth" to="$(arg INPUT_DEPTH_IMAGE)" />
-    <rosparam subst_value="true">
-      gpu: $(arg gpu)
-      model_file: $(find jsk_perception)/trained_data/pose_estimation_2d_chainermodel.pkl
-      hand:
-        enable: true
-        model_file: $(find jsk_perception)/trained_data/pose_estimation_2d_hand.chainermodel
-      with_depth: true
-      scales: [0.38]
-      stride: 8
-    </rosparam>
-  </node>
+
+  <!-- Coral TPU -->
+  <group if="$(arg coral)">
+    <node name="people_pose_estimation_2d"
+          pkg="coral_usb" type="edgetpu_human_pose_estimator.py"
+          output="screen" respawn="true">
```

Comment:
```suggestion
    <node if ="$(arg launch_coral_device)"
          name="people_pose_estimation_2d"
          pkg="coral_usb" type="edgetpu_human_pose_estimator.py"
          output="screen" respawn="true">
```

how about adding new `arg` `launch_coral_device` or something like that, and control if coral usb is connected to your pc or not?

Reply:
You mean making new rosbag file with `edgetpu_human_pose_estimator/output/poses`?
If coral:=true and launch_coral_device:=false, we cannot get peoplepose.

Reply:
For your usage, you use edgetpu topic from pr2 and we comment out this line.
So maybe your usage  is common, and other people wants to use this sample launch with pr2 or fetch.
In that case, the arg `launch_coral_device` might be useful.
I think default value of `launch_coral_device` should be `true` but we can add the arg for usage with pr2 and fetch.

Reply:
I modified for using robot.
## Usage
### PR2
```
rossetmaster pr1040
roslaunch jsk_perception sample_people_pose_estimation_2d.launch coral:=true launch_coral_device:=false
```
### fetch
```
rossetmaster fetch1075
roslaunch jsk_perception sample_people_pose_estimation_2d.launch coral:=true launch_coral_device:=false CORAL_DEPTH_IMAGE:=/head_camera/depth_registered/hw_registered/image_rect_raw CORAL_CAMERA_INFO:=/head_camera/rgb/camera_info
```
### Coral USB
```
rossetlocal
roslaunch jsk_perception sample_people_pose_estimation_2d.launch coral:=true
```
### GPU or CPU
```
rossetlocal
roslaunch jsk_perception sample_people_pose_estimation_2d.launch
```
- To visualize skeleton
`jsk_perception sample_people_pose_estimation_2d.launch` 
-> `jsk_recognition_utils sample_people_pose_3d_to_skeleton.launch`
with same params